eng/pipelines/templates/steps/build-extended-artifacts.yml

```diff
@@ -67,11 +67,11 @@ steps:
       displayName: 'Generate Docs'
       condition: and(succeededOrFailed(), ${{parameters.BuildDocs}})
       inputs:
-        scriptPath: 'scripts/devops_tasks/dispatch_tox.py'
+        scriptPath: 'eng/scripts/dispatch_checks.py'
         arguments: >-
           "$(TargetingString)"
           --service="${{ parameters.ServiceDirectory }}"
-          --toxenv=sphinx
+          --checks="sphinx"
```

Comment:
We gotta add the `env:` for `TOX_PIP_IMPL: 'uv'` So that this is screaming fast.